functions/clamp.h

```diff
@@ -0,0 +1,37 @@
+#ifndef FUNCTIONS_CLAMP_H
+#define FUNCTIONS_CLAMP_H
+
+#include "int.h"
+#include "svf.h"
+
+// Usage: Clamp<F, MIN, MAX>
+// Clamps value between MIN and MAX
+// F, A, B: INTEGER
+// return value: INTEGER
+
+template<class F, class MIN = Int<0>, class MAX = Int<32768>>
+class ClampXSVF {
```

Comment:
Needs both SVF and non-SVF versions.
(The non-svf version should do the clamp inside getInteger())

Reply:
Added, but the SVF, non-SVF still trips me up a bit so open to improvement.